### PR TITLE
Performance Profiler: Update loading image while getting LLM suggestions

### PR DESCRIPTION
**Pull request by gavande1**
## Proposed Changes

* Update loading icon with big sky rotating icon. 

## Why are these changes being made?

* As per the suggestion received in https://github.com/Automattic/wp-calypso/pull/94254#discussion_r1746681775

## Testing Instructions

* Visit following URL on calypso live link. 
```
/speed-test-tool?url=https%3A%2F%2Fwww.atliq.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NjkzNH0.uoPBbETEfzOEqTdJ_kro-zUpmvSF6vQz7JabJYumwME
````
- Click on any recommendation and notice the loading icon animation. 

https://github.com/user-attachments/assets/ca43e45a-7157-47af-b092-99a18c7f3319

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/performance-profiler/components/llm-message/index.tsx [hunk 1]
```diff
@@ -18,7 +19,7 @@ export const LLMMessage = ( { message, rotate, secondaryArea }: LLMMessageProps
 		<div className="performance-profiler-llm-message">
 			<div className="content">
 				<img
-					src={ IAIcon }
+					src={ rotate ? IALoadingIcon : IAIcon }
 					alt={ translate( 'IA generated content icon' ) }
```
Comment (gcsecsey) [inline client/performance-profiler/components/llm-message/index.tsx]:
```suggestion
					alt={ translate( 'AI generated content icon' ) }
```

**Review by gcsecsey — COMMENTED** [review]
The animation looks nice, but the copy has some typos.
image

**Review by gcsecsey — APPROVED** [review]
LGTM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16655700

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @gavande1, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `IA generated content icon`
- `Generated with IA`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16655700).

**Pull request merged**